themes/kvantum/lxqt-runner.qss

```diff
@@ -2,13 +2,9 @@ Dialog {
     background-color: transparent;
 }
 
-#commandList {
-    background-color: palette(window);
-}
-
 #panel {
     background-color:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 rgba(100, 100, 100, 80%), stop:0.11 rgba(80, 80, 80, 80%), stop:0.12 rgba(80, 80, 80, 80%), stop:1 rgba(0, 0, 0, 72%));
-    border-radius: 5px;
```

Comment:
I think you used an old branch for making the PR. All these changes to the Kvantum theme have already been applied to the latest git.

I don't understand why GitHub doesn't show a merge conflict! For example, the above lines don't exist in git. The lines below will duplicate what already exists.

Reply:
Yes it was not updated, and there was a merge conflict first because I touched by error this file, probably I messed it  up trying to fix it, if you know how to eliminate it from this PR just do it.

 I forgot: backlight slider in kde-plasma is not styled yet too.